### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alena-bartosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# 2048game
\ No newline at end of file
+# 2048game
+
+This is a console game, a copy of the famous game '2048' in my performance.
```
Comment (s373r) [inline README.md]: I guess we need to add a link (https://github.com/gabrielecirulli/2048) to the game

Comment (s373r) [reply README.md]:
```
[2048](https://github.com/gabrielecirulli/2048)
```
will be converted to: [2048](https://github.com/gabrielecirulli/2048)

Comment (s373r) [reply README.md]: **NB.**
```
[2048](https://github.com/gabrielecirulli/2048)
```
will be converted to: [2048](https://github.com/gabrielecirulli/2048)

Comment (s373r) [reply README.md]: could you please add that?

Comment (alena-bartosh) [reply README.md]: ok

Comment (alena-bartosh) [reply README.md]: done

**Pull request merged**